### ISSUE TITLE
You can add a login

### ISSUE THREAD

**Issue by halojoy** [KariNarhi/php-blog-exercise#1]
You can add a login,
so only the admin can Add Post.
Like it is now it is useless - because anyone can write posts.
Can be spammers or whatever.

The script you have done is basically very good.
A good exercise.
But you could add login and also later Comments feature.

I have added small changes to your script. A rewrite.
I changed it to use PDO database and so it can use both SQLite and MySQL.

**Comment by KariNarhi** [comment]
Yeah, I know it is far from perfect. I am more of a JavaScript guy. I just wanted to get some dust off the shelves with this PHP/SQL exercise. I might make it more robust later. I am working on my graduation thesis so I don't have time for GitHub projects at the moment.
